config/schedule.rb

```diff
@@ -0,0 +1,26 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+
+set :chronic_options, hours24: true
+
+every 1.day, at: '1:00' do
+  runner "CbdImportJob.perform_later"
```

Comment:
Guessing the error isn't just that this method should be `.perform`?

Reply:
no, it's not able to start a sidekiq instance on the server. perform_later is the perform method but puts it into a queue to run in a background job. it adds the job to the queue correctly, but then it just sits there because no sidekiq to run it.

just using perform would run i think, just not as a background job. as there's no urgency it can just wait I think